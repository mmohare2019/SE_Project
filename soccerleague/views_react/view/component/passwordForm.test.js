import "react-native";
import "react";
import FormField from "./button";
import { test, expect } from "@jest/globals";
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test("Testing form field", function () {
    const component = renderer
    .create(<FormField onPress={()=>1}/>).toJSON();
    expect(component).toMatchSnapshot();
}); 