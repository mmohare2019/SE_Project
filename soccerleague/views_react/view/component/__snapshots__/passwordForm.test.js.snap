// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing form field 1`] = `
<View>
  <Text
    style={
      Object {
        "fontSize": 16,
        "paddingTop": 10,
      }
    }
  >
     Password: 
  </Text>
  <TextInput
    autoCapitalize={false}
    onChangeText={[Function]}
    secureTextEntry={true}
    style={
      Object {
        "borderRadius": 10,
        "borderWidth": 1,
        "padding": 10,
      }
    }
  />
</View>
`;
